orchagent/fabricportsorch.cpp

```diff
@@ -101,7 +101,36 @@ FabricPortsOrch::FabricPortsOrch(DBConnector *appl_db, vector<table_name_with_pr
 
     auto debug_executor = new ExecutableTimer(m_debugTimer, this, "FABRIC_DEBUG_POLL");
     Orch::addExecutor(debug_executor);
-    m_debugTimer->start();
+    bool fabricPortMonitor = checkFabricPortMonState();
+    if (fabricPortMonitor)
+    {
+        m_debugTimer->start();
+        SWSS_LOG_INFO("Fabric monitor starts at init time");
```

Comment:
Shall we start fabric monitoring during init or Shall we give some delay as links might still be coming up ?

Reply:
the debug timer will start and polling counters, but it will skip to make any decision during the first several polls as the card might still coming up.